checks/process.go

```diff
@@ -187,6 +188,23 @@ func formatCommand(fp *process.FilledProcess) *model.Command {
 	}
 }
 
+func formatIO(fp *process.FilledProcess, lastIO *process.IOCountersStat, before time.Time) *model.IOStat {
+	diff := time.Now().Unix() - before.Unix()
```

Comment:
OK done ! However I think real_time.go already has I/O rates in its messages : ContainerRbps and ContainerWbps. They read using util/docker/cgroup/ IO function, that seems to do something similar to process.go fillFromIO function.